src/transformers/generation/logits_process.py

```diff
@@ -172,11 +172,58 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor) -> to
 
 class TemperatureLogitsWarper(LogitsWarper):
     r"""
-    [`LogitsWarper`] for temperature (exponential scaling output probability distribution).
+    [`LogitsWarper`] for temperature (exponential scaling output probability distribution) which effectively means that
+    it can control the randomness of the predicted tokens.
+
+    <Tip>
+
+    Make sure that `do_sample=True` is included in the `generate` arguments otherwise the temperature value won't have
+    any effect.
+
+    </Tip>
 
     Args:
         temperature (`float`):
-            The value used to module the logits distribution.
+            The value used to module the logits distribution. A value smaller than `1` decreases randomness (and vice
```

Comment:
```suggestion
            Strictly positive float value used to modulate the logits distribution. A value smaller than `1` decreases randomness (and vice
```

We can remove the last sentence (see below) and make it more compact this way :) Also note that `module` should be `modulate`

`make fixup` may need to be executed after this change!